assets/stylesheets/units/_buttons.scss

```diff
@@ -0,0 +1,43 @@
+.button {
+  @include bold-font(20);
+
+  -webkit-appearance: none;
+  background-color: shade($green, 12.5%);
+  border: 0;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  letter-spacing: .5px;
+  margin: 0 ($baseline-grid-unit * 6) ($baseline-grid-unit * 3) 0;
```

Comment:
First up, it's "preceded" ;)

I'll look into the spacing again. I'd got into the habit of using the atomic `$baseline-grid-unit` but will redo.

On the `* + .button`, why are we universally afraid of top margins exactly?